tests/apps/utils/helpers.go

```diff
@@ -27,3 +29,15 @@ func IsTruthy(val string) bool {
 		return false
 	}
 }
+
+// PortFromEnv returns a port from an env var, or the default value.
+func PortFromEnv(envName string, defaultPort int) (res int) {
```

Comment:
Same question as https://github.com/dapr/dapr/pull/6813/files#r1296388182.

Reply:
Doesn't seem that way, but the intention probably is to use this helper method throughout E2E tests in the future. The refactor of existing E2E tests wasn't added here.